app/models/person_escort_record.rb

```diff
@@ -7,8 +7,20 @@ def self.framework_name
     'person-escort-record'
   end
 
+  def editable
+    return false if confirmed?
+
+    move_status_editable?
+  end
+
 private
 
+  def move_status_editable?
+    return true if move.blank?
+
+    move.requested? || move.booked?
```

Comment:
This will cause one extra small query to the move on the PER serializer. If a PER is requested through a move endpoint no extra query will need to done as it will look up the cached parent.